src/libLookup/Lookup.cpp

```diff
@@ -4175,6 +4274,15 @@ bool Lookup::VerifySenderNode(const VectorOfNode& vecLookupNodes,
   return vecLookupNodes.cend() != iter;
 }
 
+bool Lookup::VerifySenderNode(const DequeOfNode& deqLookupNodes,
```

Comment:
Since this function was used for veryfing DS node, shall we rename the parameter here? Maybe just use a more generic name.